docs_autobuild.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/bash
 
 cd docs && make clean && cd ..
-poetry run sphinx-autobuild docs docs/_build/html || printf '\n\nError running docs_autobuild. \nMake sure you are using poetry >= 1.1.15. \nCheck out https://docs.robusta.dev/master/developer-guide/platform/docs-contributions.html\n'
+
+if [ "$1" == "--port" ];then
```

Comment:
@pavangudiwada instead of the if/else can we always run `poetry run sphinx-autobuild $@ docs docs/_build/html` and such that `./docs_autobuild.sh` supports all `sphinx-autobuild` flags and passes them in automatically no matter what?